landingai/pipeline/frameset.py

```diff
@@ -79,6 +80,52 @@ class Frame(BaseModel):
     metadata: Dict[str, Any] = {}
     """An optional collection of metadata"""
 
+    @property
+    def frames(self) -> List["Frame"]:
+        """Returns a list with a single frame"""
+        warnings.warn(
+            "frame.frames[x].<method> is deprecated and will be removed in future versions. "
+            "Use frame.<method> instead"
+        )
+        return [self]
```

Comment:
I don't think we need this... if search our codebase and only in two cases people have used this pattern and it is really because they did not use FramSet (i.e bad practice)... Let's refactor the examples

Reply:
Wait, we do follow semver practices... in fact I was going to tell you that this branch should move us to 0.3.x and the breaking changes should be listed on `CHANGELOG.md`.... FYI: for now we use the second digit as major because we don't consider the SDK to be a V1 release yet....

Reply:
Using the second digit as major is what I mean by not following semver. :grimacing: 

I don't want to risk breaking compatibility for end users bumping from 0.2.x to 0.3.x, because it is misleading. We suffer a lot with this kind of change in the libs we use in our backend today. :joy: 

I'll keep the compatibility layer for now, and we drop it when we reach a 1.x.x version, ok? The changes are already documented in the `CHANGELOG.md` file.